chance.js

```diff
@@ -80,6 +80,10 @@
         return options;
     }
 
+    function range(size) {
+        return Array.apply(null, Array(size)).map(function (_, i) {return i;});
+    }
```

Comment:
This is a simple implementation of lodash `range`. It may be preferrable to just pull him in, but that would be a new dependency.